coordinator/src/db/trades.rs

```diff
@@ -43,6 +50,22 @@ pub fn insert(conn: &mut PgConnection, trade: crate::trade::models::NewTrade) ->
     Ok(trade)
 }
 
+/// Returns the position by trader pub key
+#[autometrics]
+pub fn get_by_fee_payment_hash(
+    conn: &mut PgConnection,
+    payment_hash: PaymentHash,
+) -> QueryResult<Option<crate::trade::models::Trade>> {
```

Comment:
🔧 I think we should just return a `bool` and call this `is_associated_with_fee_payment` or something like that. We don't seem to be using the returned data.

Reply:
Fair enough - for me it's more a pattern that an `insert` statement should return the representation of the object that was inserted (i.e. we insert `NewTrade` and return `Trade`) - but I can also remove this for now because it's not used as you say.

Reply:
Yeah, on _inserting_ I do think you make a very good point. But here I'm advocating for replacing this _get_ (whose return value we're only using to see if it's `Some`) with a more dedicated method.

Reply:
sorry, this answer was on the wrong comment - I actioned this commend, fixup pushed now :)